CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # NHS.UK frontend Changelog
 
+## 6.0.2 - Unreleased
```

Comment:
This would be classed as a new feature so v6.1.0 would be more fitting rather than a fix version